star-ros2/src/star_gateway_bridge/include/star_gateway_bridge/message_converter.hpp

```
@@ -4,8 +4,8 @@
 // STAR Project - Texas A&M University
 // January 2026
 
-#ifndef STAR_GATEWAY_BRIDGE_INCLUDE_STAR_GATEWAY_BRIDGE_MESSAGE_CONVERTER_HPP_
-#define STAR_GATEWAY_BRIDGE_INCLUDE_STAR_GATEWAY_BRIDGE_MESSAGE_CONVERTER_HPP_
+#ifndef STAR_GATEWAY_BRIDGE__MESSAGE_CONVERTER_HPP_
+#define STAR_GATEWAY_BRIDGE__MESSAGE_CONVERTER_HPP_
```

Clang (14.0.6)

[warning] declaration uses identifier 'STAR_GATEWAY_BRIDGE__MESSAGE_CONVERTER_HPP_', which is a reserved identifier

(bugprone-reserved-identifier)